src/components/tab-controls/tab-controls.scss

```diff
@@ -33,21 +33,18 @@
 		border-color: currentColor;
 	}
 
-	:checked + &:hover {
-		border-color: $color-focus-state;
-	}
-
-	.tab-controls__input:focus + & {
```

Comment:
we actually need to _improve_ the focus state from just being purple, but it seems this code just removes it?

Reply:
This is what design asked for, yes.